frontend/src/components/notification/notification.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {useEffect} from 'react';
+import {Alert, Snackbar} from "@mui/material";
+import CheckIcon from '@mui/icons-material/Check';
+import {useAppDispatch, useAppSelector} from "../../stores/redux/hooks";
+import {selectNotification, showNotification} from "../../stores/redux/notification/notificationSlice";
+import {INotification} from "../../types/notification";
+import {NotificationType} from "../../types/notificationType";
+
+export const createNotification = (message: string, type: NotificationType): INotification => {
+    return {
+        show: true,
+        type,
+        message
+    };
+};
+
+const Notification: React.FC = () => {
+    const [message, setMessage] = React.useState<string>("");
+    const [show, setShow] = React.useState<boolean>(false);
+    const dispatch = useAppDispatch();
+    const notification = useAppSelector(selectNotification);
```

Comment:
Let's use destructuring feature
```suggestion
    const {show, message, type} = useAppSelector(selectNotification);
```